lib/node_modules/@stdlib/math/base/special/fast/max/lib/native.js

```diff
@@ -0,0 +1,59 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var addon = require( './../src/addon.node' );
+
+
+// MAIN //
+
+/**
+* Returns the maximum value.
+*
+* @private
+* @param {number} x - first number
+* @param {number} y - second number
+* @returns {number} maximum value
+*
+* @example
+* var v = max( 4.2, 3.14 );
+* // returns 4.2
+*
+* @example
+* var v = max( 0.0, -0.0 );
+* // returns 0.0
+*
+* @example
+* var v = max( 3.14, NaN );
+* // returns NaN
+*
+* @example
+* var v = max( NaN, 5.0 );
+* // returns NaN
```

Comment:
Per tests in `test.native.js`, this should be:

```suggestion
* var v = max( NaN, 5.0 );
* // returns 5.0
```